cmd/homi/docker/compose/homi.go

```diff
@@ -165,18 +172,75 @@ func (ist *Homi) init(number int, addresses, nodeKeys []string, genesis, scGenes
 			false,
 		)
 
-		staticSCNodes = strings.Replace(staticSCNodes, "0.0.0.0", s.IP, 1)
+		staticSCNNodes = strings.Replace(staticSCNNodes, "0.0.0.0", s.IP, 1)
+		ist.Services = append(ist.Services, s)
+		validatorNames = append(validatorNames, s.Name)
+	}
+
+	numSPNs := len(spnNodeKeys)
+	for i := 0; i < len(spnNodeKeys); i++ {
+		s := service.NewValidator(i,
+			"",
+			scGenesis,
+			"",
+			spnNodeKeys[i],
+			"",
+			"",
+			32323+number+numPNs+numENs+numSCNs+i,
+			8551+number+numPNs+numENs+numSCNs+i,
+			61001+number+numPNs+numENs+numSCNs+i,
+			ist.EthStats.Host(),
+			// from subnet ip 10
+			fmt.Sprintf("%v.%v", ist.IPPrefix, number+numPNs+numENs+numSCNs+i+10),
```

Comment:
Just curious. Do you know why `+10`?

Reply:
I'm not sure. Just to avoid to alloc '0' case?